projects/uitdatabank/docs/search-api/sorting.md

```diff
@@ -0,0 +1,86 @@
+# Sorting
+
+The UiTdatabank Search API provides several options to sort events, places and organizers.
+
+- `availableTo` 
+- `created`
+- `distance` 
+- `modified`
+- `score`
+
+For every sort parameter it is possible to change the sort `order`:
+
+- Ascending order \(from smallest to biggest value\) by using `asc`
+- Descending order \(from biggest to smallest value\) by using `desc`
```

Comment:
Hmm are these escapes necessary? 🤔

Reply:
Those sentences were copied from the kramdown file for documentatie.uitdatabank.be, not sure if those escapes are actually needed there